broker-core/src/main/java/io/zeebe/broker/system/configuration/BrokerCfg.java

```diff
@@ -34,6 +35,9 @@
   private RaftConfiguration raft = new RaftConfiguration();
   private List<TopicCfg> topics = new ArrayList<>();
 
+  @SerializedName("exporter")
```

Comment:
Without it, the configuration would be `[[exporters]]` instead of `[[exporter]]`, or we call the field `exporter` instead. I wanted to have the field be plural (since its a list), but the configuration section be `[[exporter]]`, since it is effectively once section per exporter. It's not super important though so if you prefer I can remove it and go with `[[exporters]]`.